packages/sdk/.eslintrc.json

```diff
@@ -15,8 +15,9 @@
     "prettier"
   ],
   "rules": {
-    "sonarjs/cognitive-complexity": "warn",
     "@typescript-eslint/no-explicit-any": "warn",
+    "@typescript-eslint/no-use-before-define": ["error", { "functions": false }],
```

Comment:
Avoid ESLint error when using a function defined further in the file. (That works well thanks to hoisting.)